data/nav.yml

```
@@ -299,6 +299,8 @@
             path: 'test-analytics/python-collectors'
           - name: 'Golang'
             path: 'test-analytics/golang-collectors'
+          - name: '.NET'
+            path: 'test-analytics/dotnet-collectors'
```

@meiqimichelle if we want to add the _beta pill_ in the left nav we can use the `pill` key.

```suggestion
            path: 'test-analytics/dotnet-collectors'
            pill: 'beta'
```

---

Ohhhhhh icic. Thanks for the explanation! I was moving too quickly in-browser 😅